sync/clients/__init__.py

```diff
@@ -2,7 +2,7 @@
 
 from sync import __version__
 
-USER_AGENT = f"Sync Library/v{__version__} (syncsparkpy)"
```

Comment:
Version information here is usually not preceded with a "v" in user agent strings.